settings.py

```diff
@@ -18,3 +18,6 @@
 USE_TZ = False
 
 INSTALLED_APPS = ("db",)
+
+ALLOWED_HOSTS = ["*"]
+DEBUG = "TRUE"
```

Comment:
I wanted to run the main.py file to check the code works and I get an error:
"CommandError: You must set settings.ALLOWED_HOSTS if DEBUG is False."
It seems that the problem was in the settings.py file, because after adding ALLOWED_HOSTS [*] everything worked correctly.
I suppose I should have removed these lines after testing?

Reply:
Yes, `settings.py` should not be present in the PR (pull request), remove it, please